src/CloudNative.CloudEvents.AspNetCore/HttpRequestExtension.cs

```diff
@@ -45,90 +22,54 @@ public static class HttpRequestExtension
         /// <param name="extensions">List of extension instances</param>

```

Comment:
nit: docs for formatter param

Reply:
Done as you raised it - but in general I haven't fixed up all the docs. That will be a future pass - possibly the next PR, possibly a little later.